lib/dropdown/DropdownCtrl.js

```diff
@@ -0,0 +1,26 @@
+import angular from 'angular';
+
+var DropdownCtrlMod = angular.module('guDropdownCtrl', []);
+
+var DropdownCtrl = DropdownCtrlMod.controller('guDropdownCtrl', [
+  '$scope',
+  function DropdownCtrl(scope){
+
+      function getClassName() {
+        return scope.state === 'open' ? 'dropdown open' : 'dropdown closed';
```

Comment:
An alternative like that would be to use `ng:class`. The way you wrote it is a bit lower level but I like it personally. It's clear and to the point and keeps the logic in the JS rather than in the template.